enos/enos-modules.hcl

```diff
@@ -77,6 +87,19 @@ module "vault_cluster" {
   vault_install_dir = var.vault_install_dir
 }
 
+module "vault_cluster_ips" {
```

Comment:
This comment applied to many of the new modules. Generally we try to include a verb that describes what the module goes. For this I'd suggest something like `vault_get_cluster_ips`